src/main/java/com/salesforce/mirus/MirusSourceTask.java

```diff
@@ -160,8 +171,10 @@ private void seekToOffsets(List<TopicPartition> partitionIds) {
         return Collections.emptyList();
       }
     } catch (WakeupException e) {
-      // Ignore exception iff shutting down thread.
```

Comment:
nitpick --  iff may have been short for if and only if  :)

Reply:
True true :)